AMAZON-INSTRUCTIONS.md

```diff
@@ -0,0 +1,33 @@
+## Instructions
+
+Add a new dependency to the build.gradle apart from the regular `purchases` dependency. This new dependency has the classes needed to use Amazon IAP:
+
+```groovy
+dependencies {
+    implementation 'com.revenuecat.purchases:purchases:5.0.0-amazon.beta.1'
+    implementation 'com.revenuecat.purchases:purchases-store-amazon:5.0.0-amazon.beta.1'
+}
+```
+
+Configure the SDK using your **RevenueCat API key specific for Amazon**:
+
+```kotlin
+Purchases.configure(AmazonConfiguration.Builder(this, "api_key").build())
+```
+
+## Migration from previous Amazon alpha versions
+
```

Comment:
Perhaps we should be more explicit here and call it "previous alpha versions of purchases-android with Amazon Store support"